core/lib/basic_types/src/prover_dal.rs

```diff
@@ -229,3 +234,121 @@ impl FromStr for GpuProverInstanceStatus {
         }
     }
 }
+
+pub struct ProverJobFriInfo {
```

Comment:
Short answer, not today. We've opted to export all the fields (1 to 1 with the table) of the table to avoid having incomplete structs (some of them only work for special cases but not for general cases).

Reply:
Ok, I agree with this. Nevertheless, I think we can keep this version until we have the `--verbose` flag implemented (there we'll know exactly the fields we need).